src/v1/users/controllers/user-controller/index.js

```diff
@@ -101,15 +87,14 @@ class UserController {
 
     try {
       const user = await this.DAO.getById(request.data.id);
-      if (username) {
-        user.username = username;
-      }
-      if (password) {
-        user.password = password;
-      }
-      if (role) {
-        user.role = role;
-      }
+
+      const filterData = { username, password, role };
+      const cleanFilterData = getCleanDataService(filterData);
+
+      Object.keys(cleanFilterData).forEach((key) => {
```

Comment:
Isn't this the same in getCleanData? Use that function

Reply:
Yes, it's similar,but not  same, getCleanData return data without undefined fields, and password to in basic view, but only in user model we transform password to salt and hash and this fields we need to save. So we need to do reassignment.
Or we can write `setCleanData(user, filterData)` for this case, but, how I understand, it's not clear for understanding way too.

Reply:
Or write service what will do generate password hash and salt and use it in user model and here.

Reply:
Hm, I will choose second variant. Are you agree?